modules/login/php/requestaccount.class.inc

```diff
@@ -1,323 +0,0 @@
-<?php
```

Comment:
@maltheism what's the reasoning behind getting rid of the individual classes/endpoints and combining them into one big authentication class with separate functions instead ?

These classes barely have anything in common, it seems more logical for separation of concerns to keep each one in it's own container.

Reply:
@ridz1208 Yah that can be done.